### PR TITLE
Update paper.bib

### PR DESCRIPTION
**Pull request by arfon**
Fixing comma

### CODE REVIEW

**Review by kthyng — CHANGES_REQUESTED** [review]
@arfon still need to remove comma from line above yes?

File: paper/paper.bib [hunk 1]
```diff
@@ -20,7 +20,6 @@ @article{pedregosa2011
   number={Oct},
   pages={2825--2830},
   year={2011},
```
Comment (kthyng) [inline paper/paper.bib:22]: extra comma here. Not sure how I can just delete it though.

Comment (arfon) [reply paper/paper.bib:22]: No, that one is OK.

**Comment by arfon** [comment]
Just to confirm, this PR _as is_ will work fine.

**Comment by rmsare** [comment]
Thanks for catching this, and sorry for the delay on my end.

**Pull request merged**